Framework/Geometry/inc/MantidGeometry/Instrument/GridDetector.h

```diff
@@ -138,7 +138,7 @@ class MANTID_GEOMETRY_DLL GridDetector : public CompAssembly,
   void initDraw() const override;
 
   /// Returns the shape of the Object
-  const std::shared_ptr<const IObject> shape() const override;
+  const std::shared_ptr<const IObject> &shape() const override;
```

Comment:
Does this not give a compiler error as they base class should now have a different return signature?

Reply:
Accidentally left the revision unstaged!